saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/mSpRL2017/mSpRLConfigurator.scala

```diff
@@ -13,4 +15,5 @@ object mSpRLConfigurator {
   val expName = "BM" //{"BM", "BM+C", "BM+C+E", "BM+C+E+I"}
   val isTrain = true
   val iterations = 50
+  val useConstraints = false
```

Comment:
yes, that was what I tried to suggest on slack, in my draft.
we can decide about the expname also in the config if you like, as follows:
```
 val expName: String = {
    if (!USE_CONSTRAINTS && !USE_EMBEDDINGS && !USE_IMAGE_EMBEDDINGS) "BM"
    else
    if (USE_CONSTRAINTS && !USE_EMBEDDINGS && !USE_IMAGE_EMBEDDINGS) "BM+C"
    else
    if (USE_CONSTRAINTS && USE_EMBEDDINGS && !USE_IMAGE_EMBEDDINGS) "BM+C+E"
    else
    if (!USE_CONSTRAINTS && !USE_EMBEDDINGS && !USE_IMAGE_EMBEDDINGS) "BM+C+E+I"
    else ""
    }
```